src/main/java/com/rakovets/course/java/core/practice/lambda_expressions/TvHelper.java

```diff
@@ -0,0 +1,63 @@
+package com.rakovets.course.java.core.practice.lambda_expressions;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TvHelper {
+
+    public static List<Tv> getTvWithNeededScreenSize(List<Tv> tvs, double screenSize) {
+        return tvs.stream()
+                .filter(tv -> tv.getScreenSize() == screenSize)
+                .collect(Collectors.toList());
+    }
+
+    public static List<Tv> getTvWithNeededBrand(List<Tv> tvs, String brand){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"